test/Macros/macro_plugin_server.swift

```diff
@@ -25,12 +25,26 @@
 // RUN:   -typecheck -verify \
 // RUN:   -swift-version 5 -enable-experimental-feature Macros \
 // RUN:   -external-plugin-path %t/plugins#%swift-plugin-server \
+// RUN:   -Rmacro-loading -verify-ignore-unknown \
```

Comment:
Am I just missing where this one is checked? Seems like only `macro-loading.txt` below is.